### PR TITLE
Improve documentations around cupy

### PR DESCRIPTION
**Pull request by beam2d**

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -94,32 +89,10 @@ That shows all logs of installation. It may helps you::
...
-  $ CUDA_PATH=/opt/nvidia/cuda pip install cupy
-
-
-.. warning::
-
-   If you want to use ``sudo`` to install CuPy, note that ``sudo`` command initializes all environment variables.
-   Please specify ``CUDA_PATH`` environment variable inside ``sudo`` like this::
-
-      $ sudo CUDA_PATH=/opt/nvidia/cuda pip install cupy
+In order to enable CUDA support, you first have to install `CuPy <https://docs.cupy.chainer.org/>`_.
```
Comment (delta2323) [inline docs/source/install.rst]: Should we install CuPy manually? If so, it is better to write it explicitly.

File: docs/source/install.rst [hunk 2]
```diff
@@ -94,32 +89,10 @@ That shows all logs of installation. It may helps you::
...
+If you also want to use cuDNN, you have to install CuPy with cuDNN support.
+See `CuPy's installation guide <http://docs.cupy.chainer.org/en/latest/install.html>`_ to install CuPy.
+Once CuPy is correctly set up, Chainer will automatically enable CUDA support.
```
Comment (delta2323) [inline docs/source/install.rst]: It is better to write how to check CUDA support is enabled in Chainer (i.e. `chainer.cuda.available` and `chainer.cuda.cudnn_enabled`)

File: docs/source/install.rst (hunk 1 above)
Comment (beam2d) [inline docs/source/install.rst]: Yes. I will revise.

File: docs/source/install.rst (hunk 2 above)
Comment (beam2d) [inline docs/source/install.rst]: OK

**Comment by beam2d** [comment]
Revised.

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**